.github/label-pr.yml

```diff
@@ -0,0 +1,12 @@
+- regExp: ".*\\.sql+$"
+  labels: ["DB"]
+- regExp: ".*\\.(cpp|h)+$"
+  labels: ["CORE"]
```

Comment:
This is wrong and reminds me that we should write this somewhere on our wiki.

By "Core" we mean everything that is:

- C++ **except** the contents of `/src/server/scripts`, which are instead "Scripts" (for historical reasons).
- DB structure changes. So SQL changes can be "Core" too.

Reply:
well, if you do DB changes that count as core too, then it will probably have a core change as well.
This was more thought of helping people to more easily understand what type of PR it is and what way it would need to be tested

Reply:
sure @Kitzunu , but can we exclude `/src/server/scripts` from applying "CORE" ?

Reply:
Dont know

Reply:
I have no idea if this will work. regEx is a regular expression